lib/src/common/model/repos_releases.dart

```diff
@@ -96,7 +96,7 @@ class Release {
   String getUploadUrlFor(String name, [String label]) =>
       "${uploadUrl.substring(0, uploadUrl.indexOf('{'))}?name=$name${label != null ? ",$label" : ""}";
 
-  bool get hasErrors => errors?.isNotEmpty;
+  bool get hasErrors => errors == null ? false : errors.isNotEmpty;
```

Comment:
👍

Reply:
I have a feeling that returning null from hasErrors was causing a problem which is why I changed it to return false. I can't quite remember the circumstances.

Reply:
I've just checked and the method createRelease has the following line:

if (release.hasErrors) {

As such hasErrors should never be null.
My experience with dart is that passing a bool with a null value always ends with someone crying.